raphtory-graphql/src/data.rs

```diff
@@ -753,3 +582,233 @@ pub(crate) mod data_tests {
         );
     }
 }
+
```

Comment:
This is a lot of commented out code, should we maybe park this in a branch instead?